test/with_api_v2/acceptance/send_test.go

```diff
@@ -104,6 +104,72 @@ receivers:
 	t.Log(co.Check())
 }
 
+// This test is similar to TestMergeAlerts except that the firing alerts have
```

Comment:
As far as I can tell this only differs in 3 lines (`am.Push`). How about deduplicating this logic and using sub-tests (via `r.Run`)?

Reply:
Sure, I've refactored the code to reduce duplication.